translate/misc/xml_helpers.py

```diff
@@ -133,3 +133,23 @@ def normalize_xml_space(node, xml_space, remove_start=False):
 
     for child in node:
         normalize_xml_space(child, remove_start)
+
+
+def reindent(elem, level=0, indent="  ", maxlevel=4):
+    """adjust indentation to match specification"""
```

Comment:
Nit: Start docstring with Titlecase and add a trailing period.

Reply:
I would also add some additional description to tell about how the arguments affect the behavior.